src/osemgrep/configuring/Engine_type.ml

```diff
@@ -1,3 +1,20 @@
 (* TODO merge with src/core/Engine_kind.ml *)
-type pro_flavor = Language_only | Intrafile | Interfile [@@deriving show]
+(* TODO Intrafile -> Intraprocedural and remove deep *)
```

Comment:
what this TODO mean?

Reply:
ah I understand the TODO now. Yes I think it would be better indeed to have
Intraprocedural | Interprocedural | Interfile. What do you think @IagoAbal ?